src/lib/New_Parser.cpp

```diff
@@ -86,8 +84,6 @@ TreeNode* New_Parser::parse(std::deque<Token>& tokenizedQ) {
 
 
 TreeNode* New_Parser::parseInclusive(std::deque<Token>& tokenizedQ) {
-    // Function to process logical operations: "&", "^", "|".
-    // Ensure both operands are of boolean type, handle the operations accordingly.
```

Comment:
Nice. I will add in comments later tonight - just want to make a comment to remember